csunplugged/config/settings/database_proxy.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""Django settings for connecting via Google Cloud SQL Proxy."""
+
+from .base import *  # noqa: F403
+
+
+# DATABASE CONFIGURATION
+# ----------------------------------------------------------------------------
+# See: https://docs.djangoproject.com/en/dev/ref/settings/#databases
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "HOST": "127.0.0.1",
+        "PORT": "5433",
+        "NAME": "csunplugged",
+        "USER": env("GOOGLE_CLOUD_SQL_DATABASE_USERNAME"),  # noqa: F405
```

Comment:
These are loaded by `. ./load-develop-deploy-envs.sh` in the `develop-deploy.sh`. The script is a secret file that looks something like this:

```
#!/bin/bash
# Used when running local Django for updating development database.

export DJANGO_SECRET_KEY="SECRET VALUE!"
export GOOGLE_CLOUD_SQL_CONNECTION_NAME="SECRET VALUE!"
export GOOGLE_CLOUD_SQL_DATABASE_USERNAME="SECRET VALUE!"
export GOOGLE_CLOUD_SQL_DATABASE_PASSWORD="SECRET VALUE!"
```